app/lib/BaseModelWithAttributes.php

```diff
@@ -1819,9 +1819,13 @@ public function getAttributeHTMLFormBundle($po_request, $ps_form_name, $pm_eleme
 				}
 			}
 			
+			$container_bundle_code = '';
 			foreach($va_element_set as $va_element) {
 				$va_element_info[$va_element['element_id']] = $va_element;
-				if (($va_element['datatype'] == 0) && ($va_element['parent_id'] > 0)) { continue; }
+				if (($va_element['datatype'] == 0) && ($va_element['parent_id'] > 0)) { 
```

Comment:
Nested containers are used for visual grouping but aren't actually part of the bundle code. It's always two level: container => sub-element. So we don't want to be appending those sub-container codes here, just the code from the top-level container.